### ISSUE TITLE
[FR] Can I module.exports multiple functions?

### ISSUE THREAD

**Issue by Borber** [chhoumann/quickadd#125]
I find that I can't export more than one function, or that exporting more than one is only useful, and using module.exports = {xxx1,xxx2} even gives me an error.

> QuickAdd: (LOG) TemplaterError: Failed to load user script /randomImage.js. Default export is not a function.

I think this makes my scripts less reusable

**Comment by Borber** [comment]
> This should be possible. Looks like Templater isn't having it. Are you placing your quicksand scripts in the same folder as your Templater script? It doesn't like that.

Yes, all my scripts are in the Templater scripts directory. I thought this was necessary, otherwise QuickAdd would not be found. It would be great if I could export more than one, I have a lot of scripts with similar functions, and even duplicate code, that I have to split into multiple.

**Comment by chhoumann** [comment]
QuickAdd just checks your vault for .js files - you can place them anywhere.

Personally, I have /scripts/templaterScripts.
In the scripts folder, I have my QuickAdd scripts. And in my nested templaterScripts folder, I have Templater scripts. This works fine.

Templater doesn't like when there are scripts with exports that it doesn't support in its Templater scripts folder. That breaks every Templater script, no matter if it has multiple exports or not.

TLDR; QuickAdd supports it. Templater doesn't. Put scripts with multiple exports outside of your Templater scripts folder and it should work.
